.github/workflows/pack-and-comment.yml

```diff
@@ -15,6 +15,13 @@ jobs:
     name: Pack and Comment
     runs-on: 'ubuntu-22.04'
     steps:
+      - name: Check event type
+        run: |
+          if [ "${{ github.event_name }}" != "pull_request" ]; then
+            echo "This workflow was not triggered by a pull request. Exiting..."
+            exit 0
+          fi
+
```

Comment:
this is to ensure that when CI is running because of e.g. a push to `main` or in the context of the merge queue we don't have this action running (and failing because it can't find or create a comment)